server/node.go

```diff
@@ -452,6 +452,7 @@ func (n *Node) startStoresScanner(stopper *util.Stopper) {
 					StartedAt:  n.startedAt,
 					RangeCount: int32(rangeCount),
 					Stats:      *stats,
+					Desc:       n.Descriptor,
```

Comment:
Do you need the NodeID separate on this structure now that you're including the full Desc?